.github/workflows/test.yml

```diff
@@ -54,3 +54,13 @@ jobs:
 
       - name: Test Documentation
         run: uv run make test_docs
+
+  test_passing:
+    if: github.repository_owner == 'viamrobotics'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Results
+        run: |
+          echo Python tests: ${{ needs.test.result }}
+          [ "${{ needs.test.result }}" == "success" ]
```

Comment:
This creates a new single test that ensures that all our tests from the test matrix have passed. This is important because we want to have tests be marked as required so that we don't auto merge a proto PR where tests have failed, but without this we need to mark all our current tests (`3.8 max`, `3.10 min`, etc) as required. Not only is this annoying, but it's error prone because certain cases might get phased out of the matrix or new cases might get added, and we could forget to update our `required` list. By having a single catch-all here and marking _that_ as required, we ensure that we're always expecting the entire matrix to pass.

This has been tested on my local branch to ensure it works.